service/r2/convert.go

```diff
@@ -195,3 +198,49 @@ func newRuleSetJSON(latest *rules.RuleSetSnapshot) ruleSetJSON {
 		RollupRules:   rrJSON,
 	}
 }
+
+// TODO(@SANDUSKY) MOVE THIS
+type idGenType int
+
+const (
+	generateID idGenType = iota
```

Comment:
NIT: Given that this is a flag, and probably always will be, this name is a little misleading. Making it sound like there could be more than 2 values.